### ISSUE TITLE
about keframe

### ISSUE THREAD

**Issue by logcy** [Kurento/bugtracker#581]
## Prerequisites
Hello, I would like to ask a question. I have a call from a to B via kurento (a > kurento > b). When B finds that there is no key frame, it will send PLI or fir RTCP packet to kurento. How does kurento forward these packets to a? Or how does kurento send the missing keyframes of B to B?
These are MANDATORY, otherwise the issue will be automatically closed.

* [x] I agree to fill this issue template.
* [x] I have read the [Troubleshooting Guide] and [Support Instructions].

[Troubleshooting Guide]: https://doc-kurento.readthedocs.io/en/latest/user/troubleshooting.html
[Support Instructions]: https://github.com/Kurento/.github/blob/master/SUPPORT.md

### About Kurento Media Server

* Kurento version:  
* Server OS:          
* Installation method:
    
  - [] [apt-get]
  - [] [Docker]
  - [] [AWS CloudFormation]
  - [] [Built from sources]

[apt-get]: https://doc-kurento.readthedocs.io/en/latest/user/installation.html#installation-local
[Docker]: https://doc-kurento.readthedocs.io/en/latest/user/installation.html#installation-docker
[AWS CloudFormation]: https://doc-kurento.readthedocs.io/en/latest/user/installation.html#installation-aws
[Built from sources]: https://doc-kurento.readthedocs.io/en/latest/dev/dev_guide.html#dev-sources

### About your Application Server

* Programming Language:  
* Kurento Client version:     

### About end-user clients

* Device(s):    
* OS(es):        
* Browser(s):  

### Run these commands

```
cat /etc/lsb-release
```

```
kurento-media-server --version
```

```
dpkg -l | grep -Pi 'kurento|kms-|gst.*1.5|nice'
```

**Comment by j1elo** [comment]
You can use Wireshark and see the packets flying in and out of Kurento. KMS forwards PLI and FIR requests to the encoder of the video (e.g. a web browser), so the sender can decide if a new keyframe should be generated. Sometimes Kurento itself acts as encoder when transcoding is enabled, in this case it is Kurento itself the one generating keyframes.

I'm closing this as it is not an issue or bug with the Kurento code.

**Comment by logcy** [comment]
thank you for your reply。
I'm not saying this is a bug in kuernto。I just have such a question and want to ask you。
In our previous packet capture, we saw that the B-end sent a PLI request to kurento, but we did not find that Kurento forwarded the PLI request to the A-end.
The call flow is as follows：  22.12 is the A,16.60 is the kurento,165.118 is the B
![image](https://user-images.githubusercontent.com/3400895/109406444-2bbfdb00-79b4-11eb-9e0c-d37aa82e8299.png)

From the packet capture, end B has sent PLI packets to KURENTO，as blow:
![image](https://user-images.githubusercontent.com/3400895/109406521-c6b8b500-79b4-11eb-9164-d724e3e4b979.png)

But I cannot find the PLI package sent from kurento to end A，as blow:
![image](https://user-images.githubusercontent.com/3400895/109406630-a2a9a380-79b5-11eb-9e15-b40923b454f7.png)


So I want to consult, why does this happen? What is the wrong configuration of my kurento? Or are the parameters incorrect during media negotiation? Or something else.
